Tests/WorkspaceTests/InitTests.swift

```diff
@@ -14,458 +14,422 @@ import Basics
+            break
         }
     }
 
-    func testInitPackageLibraryWithSwiftTestingOnly() async throws {
-        try testWithTemporaryDirectory { tmpPath in
-            let fs = localFileSystem
-            let path = tmpPath.appending("Foo")
-            let name = path.basename
-            try fs.createDirectory(path)
-
-            // Create the package
-            let initPackage = try InitPackage(
-                name: name,
-                packageType: .library,
-                supportedTestingLibraries: [.swiftTesting],
-                destinationPath: path,
-                fileSystem: localFileSystem
-            )
-            try initPackage.writePackageStructure()
+    /// Verifies the test file contents for a package.
+    private func verifyTestFileContents(
+        at path: AbsolutePath,
+        name: String,
+        hasSwiftTesting: Bool,
+        hasXCTest: Bool
+    ) throws {
+        let testFile = path.appending("Tests").appending("\(name)Tests").appending("\(name)Tests.swift")
+        let testFileContents: String = try localFileSystem.readFileContents(testFile)
+
+        if hasSwiftTesting {
+            #expect(testFileContents.contains(#"import Testing"#))
+            #expect(testFileContents.contains(#"@Test func example() async throws"#))
+        } else {
+            #expect(!testFileContents.contains(#"import Testing"#))
+            #expect(!testFileContents.contains(#"@Test func example() async throws"#))
+        }
 
-            // Verify basic file system content that we expect in the package
-            let manifest = path.appending("Package.swift")
-            XCTAssertFileExists(manifest)
-
-            let testFile = path.appending("Tests").appending("FooTests").appending("FooTests.swift")
-            let testFileContents: String = try localFileSystem.readFileContents(testFile)
-            XCTAssertMatch(testFileContents, .contains(#"import Testing"#))
-            XCTAssertNoMatch(testFileContents, .contains(#"import XCTest"#))
-            XCTAssertMatch(testFileContents, .contains(#"@Test func example() async throws"#))
-            XCTAssertNoMatch(testFileContents, .contains("func testExample() throws"))
-
-#if canImport(TestingDisabled)
-            // Try building it
-            await XCTAssertBuilds(
-                path,
-                buildSystem: .native,
-            )
-            let triple = try UserToolchain.default.targetTriple
-            XCTAssertFileExists(path.appending(components: ".build", triple.platformBuildPathComponent, "debug", "Modules", "Foo.swiftmodule"))
-#endif
+        if hasXCTest {
+            #expect(testFileContents.contains(#"import XCTest"#))
+            #expect(testFileContents.contains("func testExample() throws"))
+
+            if hasSwiftTesting {
+                // When both are present, ensure XCTest content is properly formatted
+                #expect(testFileContents.contains("import XCTest"), "XCTest import should be present")
```

Comment:
question: why does including SwiftTesting need to check for `import XCTest`?   This will be true based on line 157. and the `import Testing` is checked on line 149.